sympy/series/limits.py

```
@@ -214,9 +247,10 @@ def doit(self, **hints):
             if r is None:
                 return self
         except NotImplementedError:
-            # Trying finding limits of sequences
-            if hints.get('sequence', True) and z0 is S.Infinity:
-                trials = hints.get('trials', 5)
+            # Trying to find the limit of a sequence.
+            if sequence is None:
+                sequence = z.is_integer
+            if sequence and z0 is S.Infinity:
```

If sequence is set as True and z0 is not oo, it should raise an error.